easybuild/easyconfigs/b/BLIS/BLIS-3.0.1-GCCcore-11.2.0-amd.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'BLIS'
+version = '3.0.1'
+versionsuffix = '-amd'
+
+homepage = 'https://developer.amd.com/amd-cpu-libraries/blas-library/'
+description = """AMD's fork of BLIS. BLIS is a portable software framework for instantiating high-performance
+BLAS-like dense linear algebra libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/amd/blis/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['dff643e6ef946846e91e8f81b75ff8fe21f1f2d227599aecd654d184d9beff3e']
```

Comment:
Are you sure you don't need the broadwell dgemm patch here too?

Reply:
No, I am not sure. But personally I would only use the AMD fork on AMD systems.
Do you think it makes sense to add the broadwell here as well?

Reply:
The BLIS PR mentioned something about broadwell and zen2 having some common code. And the broadwell dgemm patch removed code that shouldn't be there so I guess it's rather safe...

Reply:
broadwell patch added